databroker/assets/sqlite.py

```diff
@@ -255,14 +255,18 @@ def find_one(self, query):
         # it up in an older one.
         for select in [SELECT_RESOURCE, OLD_SELECT_RESOURCE]:
             with cursor(self._conn) as c:
-                c.execute(select, (query['uid'],))
-                raw = c.fetchone()
+                raw = None
+                try:
+                    c.execute(select, (query['uid'],))
+                    raw = c.fetchone()
+                except sqlite3.OperationalError:
+                    pass
```

Comment:
Could this be `continue`?

Reply:
...and then
```
else:
    break
```

I think that would make this easier to follow, rather than using the `None` as a proxy for "Did it raise?"

Reply:
Maybe, are we expecting a `None`? It seems that the `if raw is None` is original to the code. See: https://github.com/NSLS-II/databroker/blame/2ff5d8a2cae8c76651b3924dc7e8b2b5ec374072/databroker/assets/sqlite.py